spec/system/sanity_check_spec.rb

```diff
@@ -5,4 +5,14 @@
   it "runs rspec" do
     true
   end
+
+  describe 'index' do
```

Comment:
Putting this here as a first pass was fine, but for the PR could you build out the test more? 
- Put it in a file with a name that aligns with what's being tested (if we have another system spec we don't have to keep the "sanity check" spec)
- Be sure that you are testing what you say you are testing. So, does this test prove that the data has been indexed to solr? Or does it prove that there is a static page that says the files have been indexed?

Reply:
I agree with Max here, though I'd take it a step further and question the existence of spec/system as a spec directory at all. Why does this exist? You are testing Rspec in this file... that is not what you came here to do ;-)

Reply:
We usually use system specs for integration-type specs, or testing "from the outside in," using Capybara to drive a browser. There are some examples in Blacklight. The initial "sanity check" spec was just for testing the setup of Capybara and its integration with CircleCI, which is now working fine, so we can get rid of the "Rspec test" now that it's working.